network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -404,3 +405,45 @@ class GeneralProductPage(ProductPage):
 
     class Meta:
         verbose_name = "General Product Page"
+
+
+class BuyersGuidePage(FoundationMetadataPageMixin, Page):
+    max_count = 1
+    template = 'buyersguide/home.html'
+    subpage_types = [SoftwareProductPage, GeneralProductPage]
+
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        if request.user.is_authenticated:
+            products = ProductPage.objects.all()
+        else:
+            products = ProductPage.objects.live()
+
+        # TODO:
+        # Sort the products by their creepiness level. Example code below taken
+        # from buyersguide/views.py
+        # def get_average_creepiness(product_dict):
+        #     try:
+        #         votes = product_dict['votes']
+        #         creepiness = votes['creepiness']
+        #         avg = creepiness['average']
+        #         return avg
+        #     except TypeError:
+        #         pass
+        #     except AttributeError:
+        #         pass
+
+        #     return 50
+        # products = cache.get_or_set(
+        #     'sorted_product_dicts',
+        #     lambda: sorted([p.to_dict() for p in Product.objects.all()], key=get_average_creepiness),
+        #     86400
+        # )
+
+        context['categories'] = BuyersGuideProductCategory.objects.all()
+        context['products'] = products
+        context['web_monetization_pointer'] = settings.WEB_MONETIZATION_POINTER,
```

Comment:
hm, we should figure out what this is and what we use it for. @anilkanji any idea what "web monetization" is in the context of PNI?

Reply:
`web_monetization_pointer` adds a meta element to the page.

```
  {% if web_monetization_pointer %}
    <meta name="monetization" content="{{ web_monetization_pointer }}">
  {% endif %}
```

Dunno if we need it or not, so I opted to keep it. Will keep this open until we get a better answer.